src/handler.ts

```diff
@@ -23,17 +23,65 @@ const headers = {
 export const handler = async (event: APIGatewayEvent, _: Context): Promise<APIGatewayProxyResult> => {
   logger.info('handler: triggered');
 
+  const requestType = event.queryStringParameters?.type;
+
+  if (requestType === 'plate') return handlePlateRequest(event);
+
+  if (requestType === 'letter') return handleLetterRequest(event);
+
+  logger.error('handler: no correct query param given');
+  return { headers, statusCode: 500, body: null };
+};
+
+export const handlePlateRequest = async (event: APIGatewayEvent, type = 'plate') => {
```

Comment:
Duplicate of the magic factory strings... perhaps:

```suggestion
  if (requestType === 'plate') return handlePlateRequest(event, requestType);

  if (requestType === 'letter') return handleLetterRequest(event, requestType);

  logger.error('handler: no correct query param given');
  return { headers, statusCode: 500, body: null };
};

export const handlePlateRequest = async (event: APIGatewayEvent, type: string) => {
```

Reply:
(and the other handler similarly ` .... type: string)`